.github/workflows/copywrite.yml

```
@@ -0,0 +1,19 @@
+name: Check Copywrite Headers
+
+on:
+  push: {}
+
+jobs:
+  copywrite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - uses: hashicorp/setup-copywrite@867a1a2a064a0626db322392806428f7dc59cb3e # v1.1.2
```

```suggestion
      - uses: hashicorp/setup-copywrite@v1
```

IIRC we're allowed to use versions for our actions.